src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,65 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                id:
+                    description: Annotation ID
+                    type: string
+                access:
+                    $ref: ../misc/access-level.v1.yaml
+                hasParent:
+                    description: If the annotation has a parent annotation
+                    type: boolean
+                    default: false
+                document:
+                    title: Annotated document
+                    type: object
```

Comment:
This object is made up of values already available in a hypothesis response. It should give us everything we need to serve the designs.